StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentMethodFormViewController.swift

```diff
@@ -54,6 +54,16 @@ class PaymentMethodFormViewController: UIViewController {
             if case .external(let paymentMethod) = paymentMethodType {
                 return .external(paymentMethod: paymentMethod, billingDetails: params.paymentMethodParams.nonnil_billingDetails)
             }
+            
+            if case .instantDebits = paymentMethodType {
+                // We create the final payment method in the bank auth flow, therefore treating the Instant Debits
+                // result like a saved payment option.
+                guard let paymentMethod = instantDebitsFormElement?.getLinkedBank()?.paymentMethod.decode() else {
+                    return nil
+                }
+                return .saved(paymentMethod: paymentMethod, confirmParams: nil)
```

Comment:
There's maybe something smelly about this being inside the `if let params = form.updateParams(..)` block when we ignore the params. We could use the `params.instantDebitsLinkedBank` instead of this but I'm not sure that gets at the crux of the issue. 

I'll leave this half-baked comment here but it's not at all blocking b/c I'm not sure what the problem is, if any.

Reply:
True, that is weird. I’ll use `params.instantDebitsLinkedBank` for now, which seems like a marginal improvement.